docs/components/tools.md

```diff
@@ -436,6 +436,8 @@ Flags:
                              Path to YAML file that contains object store
                              configuration. See format details:
                              https://thanos.io/tip/thanos/storage.md/#configuration
+      --output="table"       Output format for result. Currently supports table,
+                             cvs, and, tsv.
```

Comment:
```suggestion
                             csv, and tsv.
```

Reply:
or maybe make it "technical" and leave out the and altogether?

```suggestion
                             csv, tsv.
```

Reply:
> or maybe make it "technical" and leave out the and altogether?

this one